toplevel/role/role.go

```diff
@@ -303,3 +304,18 @@ func addOptionalOidcDefaults(instance string, roles []entry) {
 		}
 	}
 }
+
+func ValidateRoleUniqueness(desiredRoles []entry, toplevel string) error {
```

Comment:
```suggestion
func validateRoleUniqueness(desiredRoles []entry, toplevel string) error {
```

Reply:
small nit so a function unique to role isn't exported by package

Reply:
TIL!